### PR TITLE
Add - Home Page

### PR DESCRIPTION
**Pull request by Vishnu-Aithal**
add Home-page
Preview - https://deploy-preview-1--vishnu-store.netlify.app/

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,49 @@
+.banner {
+  display: flex;
+  gap: 0.5rem;
```
Comment (Aayushjain-code) [inline style.css:3]: The gap is very much trying to reduce it 0.2rem would work.

**Review by Aayushjain-code — APPROVED** [review]

File: style.css (hunk 1 above)
Comment (Vishnu-Aithal) [reply style.css:3]: Thanks For Reviewing.
that space is due to justify contents space around, but it is required for responsiveness.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+          >
+        </div>
+      </div>
+      <div class="card p-2 shadow-sm">
+        <div class="card__badge bg-yellow p-1 br-2">70% Off</div>
+        <div class="card__img-wrapper">
+          <img
+            class="card__img"
+            src="/assets/images/casual-set-1.jpg"
+            alt=""
```
Comment (Developers-codz) [inline index.html:116]: provide alt and srcset value

**Review by Developers-codz — APPROVED** [review]
Else Look good to me .

File: index.html (hunk 2 above)
Comment (Vishnu-Aithal) [reply index.html:116]: Ok thanks for the review

**Pull request merged**